conf/application.conf

```diff
@@ -126,6 +126,7 @@ features {
   allowDirectDebits.enabled = true
   staticDate.enabled = true
   vatCertificate.enabled = true
+  vatOptOutService.enabled=true
```

Comment:
Could you also make an `app-config-base` PR to add the new config there and blank out the `vat-opt-out-frontend` host, and an `app-config-production` PR to set this feature switch to false.